public/src/js/widgets/columns/fronts-config.html

```diff
@@ -279,6 +279,11 @@
             <label for="frontsToolSettings.displayEditWarning">Display edit warning</label>
             <input id="hideShowMore" type="checkbox" data-bind="checked: meta.frontsToolSettings.displayEditWarning" />
 
+            <!-- ko if: meta.type() === 'scrollable/small' -->
```

Comment:
I've limited it to scrollable small here - I guess we'll want it for scrollable medium too? Will have another PR for some more changes to make in this file, so could update then.